src/CSI.IBTA.Shared/Types/TaskResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace CSI.IBTA.Shared.Types
```

Comment:
not using and could be removed

Reply:
it is being used. I suppose that you were talking about FileUtils

Reply:
Yeah, I see the call of CreateEmployer. 
The classes GenericHttpResponse, GenericInternalResponse, TaskResult, GenericResponse are confusing and doing the same task actually. 

Can you leave one of them and drop the rest? Does not matter which class but lets make it universal and do not maintain that zoo. 

Current PR could be merged, I am not rejecting it. 
After the merge, lets do one more PR, that solves the responses issue. 

cc @mykytatishkin @VidasBui